News-Android-App/src/main/java/de/luhmer/owncloudnewsreader/NewsDetailFragment.java

```diff
@@ -304,7 +304,13 @@ private void applyWebSettings() {
         webSettings.setDomStorageEnabled(true);
         webSettings.setJavaScriptCanOpenWindowsAutomatically(false);
         webSettings.setSupportMultipleWindows(false);
-        webSettings.setSupportZoom(false);
+
+        boolean zoomEnabled = mPrefs.getBoolean(SettingsActivity.CB_DETAILED_VIEW_ZOOM, false);
+        webSettings.setSupportZoom(zoomEnabled);
+        webSettings.setBuiltInZoomControls(zoomEnabled);
+        webSettings.setDisplayZoomControls(false);
```

Comment:
I haven't really seen apps using these native (and pretty ugly, at least on my phone) native zoom elements and it felt a bit too much to add an extra setting just for this, but happy to change it

Reply:
I agree 😀 

For anyone else wondering what it looks like.. here is a screenshot: 

<img height="500" alt="Screenshot_20251226-104527" src="https://github.com/user-attachments/assets/b3edd246-9e72-4ca6-8167-30a87ae81ab5" />